src/config.js

```diff
@@ -0,0 +1,391 @@
+   * BE26B07B: tpubD9ArfXowvGHnuECKdGXVKDM...FxPa
+   * 4369050F: tpubD8NXmKsmWp3a3DXhbihAYbY...9C8n
+   * 
+   * @param {string} configString valid coldcard config string
+   * @returns {MultisigWalletConfig} new instance of MultisigWalletConfig
+   * @example
+   * import { MultisigWalletConfig } from "unchained-wallets";
+   * 
+   * const config = MultisigWalletConfig.fromColdCardConfig(ccFileData);
+   * console.log(JSON.parse(config.toJSON()))
+   * 
+   * {
+   *    name: MeMyself,
+   *    quorum: {
+   *      totalSigners: 4,
+   *      requiredSigners: 2,   
+   *    },
+   *    addressType: 'P2WSH',
+   *    network: 'testnet',
+   *    extendedPublicKeys: [
+   *      {
+   *        xfp: 'D0CFA66B',
+   *        xpub: 'tpubD9429UXFGCTKJ9NdiNK4rC5...DdP9'
+   *      },
+   *      {
+   *        xfp: '8E697B74',
+   *        xpub: 'tpubD97nVL37v5tWyMf9ofh5rzn...XgSc',
+   *      },
+   *      {
+   *        xfp: 'BE26B07B',
+   *        xpub: 'tpubD9ArfXowvGHnuECKdGXVKDM...FxPa',
+   *      },
+   *      {
+   *        xfp: '4369050F',
+   *        xpub: 'tpubD8NXmKsmWp3a3DXhbihAYbY...9C8n',
+   *      }
+   *    ]
+   * }
+   */
+  static fromColdCardConfig(configString) {
+    if (typeof configString !== 'string') {
+      throw new TypeError(
+        `ColdCard config must be a string, instead received ${typeof configString}`
+      );
+    }
+    
+    const options = {};
+    // split the text by line
+    const lines = 
+      configString
+        .split('\n')
+        // formatting
+        .map(line => line.trim())
+        // ignore comments
+        .filter(line => line[0] !== '#' && line.length);
+
+    // for each line separate key and values by colon
+    lines.forEach(line => {
+      // trim white space and set to lowercase
+      const [key, value] = line.split(':').map(str => str.trim());
+      // handle recognized keys (name, format, derivation, and policy)
+      switch (key.toLowerCase()) {
+        case 'name':
+          options.name = value;
+          break;
+        case 'format':
+          options.addressType = value.toUpperCase();
+          break;
+        case 'policy': {
+          let [m, n] = value.split(' of ');
+          m = Number(m);
+          n = Number(n);
+          // confirm we have valid numbers for m and n
+          assert(!isNaN(m) && !isNaN(n), 'Invalid policy in coldcard config');
+          options.quorum = {
+            requiredSigners: m,
+            totalSigners: n,
+          } 
+          break;
+        }
+        default: {
+          // handle possible xfp: xpub pairs from config
+          try {
+            // will throw if not a valid 
+            ExtendedPublicKey.fromBase58(value);
+            
+            // check if the key is a valid fingerprint
+            validateXFP(key);
+
+            // get the network from xpub prefix
+            const prefix = value.slice(0, 4)
+            let network;
+            if (prefix === 'xpub') {
```

Comment:
prefix can be in x/Y/Zpub for mainnet and t/U/Vpub for testnet

Reply:
Thanks. Will add a utility that should probably also go into uc-bitcoin I think